scripts/exportexcel.py

```
@@ -0,0 +1,98 @@
+import sys
+from openpyxl.utils import get_column_letter
+
+# Get the current working directory
+current_dir = os.getcwd()
+
+# Add it to sys.path
+if current_dir not in sys.path:
+    sys.path.append(current_dir)
+
+
+def prettify_json(to_prettify):
```

Ruff (0.12.2)

Missing return type annotation for public function `prettify_json`

(ANN201)